components/backup/src/writer.rs

```diff
@@ -19,9 +20,13 @@ use crate::{Error, Result};
 pub struct BackupWriter {
     name: String,
     default: SstWriter,
-    default_written: bool,
+    default_kvs: u64,
+    default_bytes: u64,
+    default_checksum: u64,
     write: SstWriter,
-    write_written: bool,
+    write_kvs: u64,
+    write_bytes: u64,
+    write_checksum: u64,
```

Comment:
Can we collect the `(SstWriter, kvs, bytes, checksum)` into a single struct? There's lots of code duplication